tests/next_tests/unit_tests/iterator_tests/transforms_tests/test_collapse_tuple.py

```diff
@@ -14,47 +14,64 @@
 
 import pytest
 
-from gt4py.next.iterator import ir
+from gt4py.next.iterator import ir, ir_makers as im
 from gt4py.next.iterator.transforms.collapse_tuple import CollapseTuple
 
 
-def _tuple_get(i: int, t: ir.Expr):
-    return ir.FunCall(fun=ir.SymRef(id="tuple_get"), args=[ir.Literal(value=str(i), type="int"), t])
+def _tup_of_size_2(first=im.ref("first_elem"), second=im.ref("second_elem")) -> ir.Expr:
```

Comment:
I think this function is superfluous by now and all calls can be replaced by just `im.make_tuple("first", "second")`.